textile/features.textile

```diff
@@ -93,7 +93,8 @@ h3(#restclient). RestClient
 * @(RSC21)@ @RestClient#push@ attribute provides access to the @Push@ object that was instantiated with the @ClientOptions@ provided in the @RestClient@ constructor
 * @(RSC22)@ @RestClient#batch@ attribute provides access to the @BatchOperations@ object that was instantiated with the @ClientOptions@ provided in the @RestClient@ constructor
 * @(RSC7)@ Sends REST requests over HTTP and HTTPS to the REST endpoint @rest.ably.io@
-** @(RSC7a)@ The @X-Ably-Version@ HTTP header must be included in all REST requests to Ably endpoints. The value to be sent is defined by "@CSV2@":#CSV2.
+** @(RSC7a)@ This clause has been replaced by "@RSC7e@":#RSC7e. It was valid up to and including specification version @2.0@.
+** @(RSC7e)@ The @X-Ably-Version@ HTTP header must be included in all REST requests to Ably endpoints. The value to be sent is defined by "@CSV2@":#CSV2, except in the case where the user directly calls @RestClient#request@, in which case the value to be sent is defined by "@RSC19f1@":#RSC19f1.
```

Comment:
Mind if I ask as I'm out of the loop. What is the thinking behind adding new spec items and deprecating old ones instead of just updating them?

Reply:
We've been doing this for a while now and it's documented [here](https://github.com/ably/specification/blob/main/CONTRIBUTING.md#features-spec-points), basically the idea is that if someone finds a spec point referenced outside of this document then they can search for it and see when it was removed + what it was replaced by, rather than the search showing up nothing

Reply:
Going to resolve this comment so I can merge (https://github.com/ably/specification/pull/153#issuecomment-1840779790).